backend/composer/enums.py

```diff
@@ -78,7 +78,7 @@ class ExportRelationships(models.TextChoices):
     hasSomaPhenotype = "hasSomaPhenotype", "SomaPhenotype"
     hasAlert = "hasAlert", "Alert"
     hasSomaLocatedIn = "hasSomaLocatedIn", "Soma"
-    hasProjectionPhenotype = "hasProjectionPhenotype", "ProjectionPhenotype"
+    hasProjectionPhenotype = "hasProjection", "ProjectionPhenotype"
```

Comment:
question (non-blocking): Shouldn't it be be hasProjectionLaterality? src: https://docs.google.com/spreadsheets/d/1n4jWZrJmKmvX2BqnKYwwbn4KK-j3q6pIAHWICborH5I/edit?gid=508127170#gid=508127170

Reply:
https://metacell.slack.com/archives/C0477SPSQQK/p1721042584112299?thread_ts=1720622046.882739&cid=C0477SPSQQK